addons/l10n_mx/models/account_tax.py

```diff
@@ -31,3 +32,13 @@ class AccountTax(models.Model):
     def _compute_l10n_mx_tax_type(self):
         for tax in self:
             tax.l10n_mx_tax_type = 'iva' if tax.country_id.code == 'MX' else False
+
+    def _prepare_tax_totals(self, base_lines, currency, company, tax_lines=None):
+        taxes_result = super()._prepare_tax_totals(base_lines, currency, company, tax_lines)
+        taxes_result['local_tax_edit_mode'] = False
+        for base_line in base_lines:
+            tax_details_results = self._prepare_base_line_tax_details(base_line, company)
+            if any(tax['tax'].l10n_mx_tax_type == 'local' for tax in tax_details_results['taxes_data']):
+                taxes_result['local_tax_edit_mode'] = True
+                break
```

Comment:
Should do the same and avoids the `break` statement. Tell me if i'm wrong tho.

```suggestion
        while not taxes_result['local_tax_edit_mode']:
            tax_details_results = self._prepare_base_line_tax_details(base_lines.pop(), company)
            if any(tax['tax'].l10n_mx_tax_type == 'local' for tax in tax_details_results['taxes_data']):
                taxes_result['local_tax_edit_mode'] = True
```

Reply:
it's not a big deal here. I added it to avoid continuing the iterations for no reason when I reach the answer.